### ISSUE TITLE
About:  4th Face Anti-spoofing Workshop and Challenge@CVPR2023, Wild Track

### ISSUE THREAD

**Issue by Xianhua-He** [deepinsight/insightface#2261]
1. Whether the dev data(dev.txt) can be added to the training, such as self-supervised pre-training or unsupervised？
2. Whether the final calculation process is to find the best threshold on the prediction score of the dev data(dev.txt) and  then use the defined threshold to calculate the ACER on the test set (test.txt).

**Comment by oggyfaker** [comment]
I have some questions need  to be answered ! 
3. The rule of this contest need only single model, but if loading more than 1 checkpoint for single model is allowed ? 
4. About the pretrained weights all team can't use: 'Any extra data or pretrained model trained from extra data cannot be used'. By the standard challenge i joined, i know this rule means "Not using any pretrained model trained from extra data related this challenge (something like pretrained of Anti-Spoofing face or relate task of face), but ImageNet is allowed". If the host only want model trained from scratch, please note it before competition beginning. Not until last week you just notice ! 
I hope the host consider this rule, not only me, all team spend a lot of time and resource for training and evaluating.

**Comment by nttstar** [comment]
@Xianhua-He 
1. it depends on how you use it, we will check.
2. yes.

**Comment by nttstar** [comment]
@oggyfaker 
3. the total FLOPs need to match our rule(<=5G FLOPs)
4. ImageNet is not allowed according to our rules, it is also an external dataset.

**Comment by nttstar** [comment]
If ImageNet can be used, why can't MS-COCO be used? There're various subsets such as ImageNet-1K, ImageNet-11K, and ImageNet-21K, and there are also different clean versions, which bring another kind of unfairness.

**Comment by AlexanderParkin** [comment]
Will labels for dev.txt be provided before the end of the competition for local validation?

**Comment by nttstar** [comment]
@AlexanderParkin No

**Comment by stormand** [comment]
So unannotated dev data(dev.txt) can be used for training? @nttstar

**Comment by nttstar** [comment]
@stormand Yes, and depends on how you use it, we will check.

**Comment by Amadeus-AI** [comment]
What will the final ranking look like?
If you didn't submit code (other than top3), will you still appear on the final ranking table? (e.g. on paper)
@nttstar
